exampleCourse/questions/template/checkbox/random-prompt/server.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+
+def is_prime(a):
+    if a == 1:
+        return False
+    return all(a % i for i in range(2, a))
+
+
+def generate(data):
+    concept = random.choice(["prime", "even", "odd"])
+    data["params"]["concept"] = concept
+
+    is_odd_correct = str(concept == "odd").lower()
+    is_even_correct = str(concept == "even").lower()
```

Comment:
Nit: lower() is not necessary here: https://github.com/PrairieLearn/PrairieLearn/blob/2985819e6f7f01980951d473ee297ae68c5e814c/apps/prairielearn/python/prairielearn.py#L607

Reply:
This code no longer exists.